pkg/pdapi/pdetcd.go

```
@@ -93,6 +95,14 @@ func (c *pdEtcdClient) PutKey(key, value string) error {
 	return err
 }
 
+func (c *pdEtcdClient) PutTTLKey(key, value string, ttl int64) error {
+	ctx, cancel := context.WithTimeout(context.Background(), c.timeout)
+	defer cancel()
+	lease, _ := c.etcdClient.Grant(ctx, ttl)
```

Why ignore the error here?